composer.json

```diff
@@ -34,11 +34,15 @@
     "psr/log":                      "~1.0"
   },
   "require-dev": {
+    "symfony/asset":          "~2.7|~3.0|~4.0",
     "symfony/phpunit-bridge": "~2.7|~3.0|~4.0",
     "symfony/twig-bundle":    "~2.7|~3.0|~4.0",
     "symfony/var-dumper":     "~2.7|~3.0|~4.0",
     "symfony/yaml":           "~2.7|~3.0|~4.0"
   },
+  "suggest": {
```

Comment:
why would you like to have it in "suggest" also?

Reply:
symfony/asset is used in debug panel, which is usually used during development
But seldom in production, where composer is installed with `--no-dev` option - symfony/asset won't be installed, but it would be suggested

Like this 
https://github.com/symfony/config/blob/master/composer.json#L34 
https://github.com/symfony/console/blob/master/composer.json#L36